appveyor.yml

```diff
@@ -0,0 +1,19 @@
+install:
+  - cinst -y jdk7
+  - cinst -y ant -ignoreDependencies
+  - set JAVA_HOME=C:\Program Files\Java\jdk1.7.0
+  - set PATH=%JAVA_HOME%\bin;%PATH%
+  - set BUCK_EXTRA_JAVA_ARGS=-Djna.nosys=true
+  - python --version
+  - ant -version
+
+
+build_script:
+  - "bin\\buck build buck"
```

Comment:
Yea, I barely know how to set up the android sdk on linux, much less windows. Perhaps https://github.com/googlesamples/android-ndk/blob/master/appveyor.yml is a good place to start?